controllers/exec/execPod_test.go

```diff
@@ -302,7 +302,7 @@ func Test_execPod_Exec(t *testing.T) {
 		// then
 		require.Error(t, err)
 		assert.ErrorIs(t, err, assert.AnError)
-		assert.Empty(t, actualOut)
+		assert.Equal(t, "", actualOut.String())
```

Comment:
```suggestion
		assert.Empty(t, actualOut.String())
```
Equal is unnecessary here.